.github/workflows/python-publish-release.yml

```diff
@@ -1,90 +0,0 @@
-name: RELEASE BUILD - PyPI 📦 Distribution
-
-on: [push, pull_request, release, workflow_dispatch]
-
-jobs:
-  build_wheels:
-    name: Build wheels on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Set up MSVC x64
-        if: matrix.os == 'windows-latest'
-        uses: ilammy/msvc-dev-cmd@v1
-
-      - name: Set up QEMU
-        if: runner.os == 'Linux'
-        uses: docker/setup-qemu-action@v3
-        with:
-          platforms: all
-
-      - name: Build wheels
-        uses: pypa/cibuildwheel@v2.20.0
-        env:
-          CIBW_ARCHS_MACOS: "x86_64 universal2 arm64"
-          CIBW_ARCHS_LINUX: "x86_64 i686 aarch64" # ppc64le s390x really slow
-          CIBW_ARCHS_WINDOWS: "AMD64" # ARM64  Seems ARM64 will rebuild amd64 wheel for unknow reason.
-          CIBW_BUILD: "cp38-* cp39-* cp310-* cp311-* cp312-*"
-          CIBW_SKIP: ""
-        with:
-          package-dir: bindings/python
-
-      - uses: actions/upload-artifact@v4
-        with:
-          path: ./wheelhouse/*.whl
-          name: artifacts-${{ matrix.os }}
-
-      - name: Check binaries (Windows)
-        if: matrix.os == 'windows-latest'
-        run: |
-          python3.exe suite/check_wheel_bin_arch.py ./wheelhouse/
```

Comment:
Ah sorry. One last thing. Please add this step as well. Those scripts are somewhat dirty. But we had quite some issues that the wheel was build incorrectly, and contained the wrong binary.

Reply:
I guess those scripts where needed before because of the hacks that were present in the `setup.py` + the workflow was not testing them properly. That is not really needed anymore because we do build on specific platforms/architectures and run tests, makes sense?

Reply:
Ok, nice. Please add the link to the test release later. I'll check them anyways. Last release it was a bad awakening :D